Formula/ghostscript.rb

```diff
@@ -49,7 +49,6 @@ def install
       --disable-gtk
       --disable-fontconfig
       --without-libidn
-      --with-system-libtiff
```

Comment:
Can you please explain why you removed this option?

Reply:
With this option, configure gives an error:
```
configure: error: Mixing local libtiff with shared libjpeg not supported
```

Reply:
Thank you for the explanation, much appreciated.

Reply:
Can't we define libjpeg as local too?

Reply:
I don't see flags like `--with-system-libjpeg` or similar.